packages/moonstone/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 ### Added
 
 - `moonstone/Panels` slot `<controls>` to easily add custom controls next to the Panels' "close" button
-- `moonstone/Tooltip` and `moonstone/TooltipDecorator` support for relative positioning via the `tooltipRelative` prop. This is an advanced feature and requires a container with specific rules. See documentation for details.
```

Comment:
I was really hoping that these two extra sentences could stay in the changelog. I think their purpose is important to illustrate that one cannot just turn it on blindly and expect that it does what they want automatically.

Even if it doesn't return, you'll need to remove the trailing period from the line, now that it's just one sentence.

Reply:
I removed the extra content because we typically do not expound upon the features added within the change log and rely on the developer to check the specific API documentation for details.

If you feel the information is critical, please add it back in.

Reply:
I feel this one, being less automatic than our typical feature additions, requires additional detail in an attempt to prevent usage frustration and potential bug reports. "I turned on relative positioning and my tooltip doesn't show or shows in the screen corner." - "You didn't set it up correctly. Read the docs." Calling out the departure from our usual fully-automatic Boolean props seems warranted in this case. I'll restore the last two sentences, leaving your updated version as sentence one.